projects/npm-tools/packages/npm-scripts/src/index.js

```diff
@@ -30,7 +30,10 @@ module.exports = async function () {
 		},
 
 		storybook() {
-			require('./scripts/storybook')();
+
+			// Storybook is temporarily disabled until it supports webpack 5
```

Comment:
Are we using it in DXP currently?

Reply:
I think in two places.

We [discussed the trade-off in Slack](https://liferay.slack.com/archives/DFUBFSDGT/p1605183277123500). My thought is that it's an actively maintained project, so we can be optimistic that this is a temporary measure — they will upgrade to webpack 5 (and by the way, we should check their issue tracker to add our `:+1:` and track the status).

Two observations though:

- We should log something to the console here with a link to where all this is fully explained.
- (Also as discussed on Slack) If we get pushback for dropping support, there are workarouds, like doing a separate build that bundles webpack v4 "inside" without adding it to the Yarn workspace, effectively hiding the dependency. It would be best if we could avoid that complexity/hassle though.

Reply:
![okay_gosling](https://user-images.githubusercontent.com/905006/99078715-bb34b000-25b6-11eb-8bf0-4cdd575da234.gif)

Reply:
Not sure about the console thing. I mean, I like the idea, but with the current portal's build nobody will see it. 

But it's OK, I'll add the log linking to this comment. It's better than nothing I guess :-). I will also notify the team using storybook about this. 

Nevertheless, keep in mind that we are not releasing anything yet, so this won't make effect in portal until we release a new version of `npm-tools`...

Reply:
> Not sure about the console thing. I mean, I like the idea, but with the current portal's build nobody will see it.

The team that uses storybook runs it by hand, so they will see it (it is not part of the build process). It will be very helpful to them to understand why their stuff has suddenly stopped working.

Proof that they care about it: [this PR from 13 days ago unbreaking it](https://github.com/liferay/liferay-frontend-projects/pull/221).

> Nevertheless, keep in mind that we are not releasing anything yet, so this won't make effect in portal until we release a new version of `npm-tools`...

Obviously, no effect until you take it out of draft and merge it. Note that @bryceosterhaus is [probably going to do a release soon](https://github.com/liferay/liferay-frontend-projects/pull/234).

Reply:
Perfect then :+1: